installer/debian/postinst

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 
-adduser --system --home /var/lib/algorand --no-create-home algorand >/dev/null
-chown -R algorand /var/lib/algorand
+adduser --system --group --home /var/lib/algorand --no-create-home algorand >/dev/null
+# prior versions didn't create this group and adduser above won't create it if the algorand user exists
+getent group algorand >/dev/null || groupadd algorand
```

Comment:
Use `addgroup --system algorand` so that it gets a low GID (instead of next-available-after-1000).